src/InputBar.js

```diff
@@ -69,17 +70,17 @@ export default class InputBar extends React.Component {
                             <button class="btn btn-primary" type="submit"
                                 title="Roll"
                                 disabled={!formulaIsRollable(this.props.formula)}
-                                onClick={this.handleRollClick}>
+                                onClick={this.handleRollClick}><span className="sr-only">Submit</span>
                                     <FontAwesomeIcon icon={faDice} />
                             </button>
                         </div>
                     </div>
                 </div>
-                {Object.values(this.dialogs).map(dialog => (
-                    <div class={this.state.openDialog === dialog ? "" : "d-none"}>
+                {Object.values(this.dialogs).map(dialog => this.state.openDialog === dialog ? (
+                    <div>
                         {dialog.content}
                     </div>
-                ))}
+                ) : null)}
```

Comment:
A previous version returned `null` instead of setting the class to `d-none`.  This results in losing the state of the formula tools dialog.

Reply:
well have to chat about this one, i'm not seeing the difference.

Reply:
I just tested it on my machine and it seems to be keeping state even after closing and reopening the dialog.  Maybe this was caused by  different issue before.

Reply:
Wait, no... I was looking at the wrong branch.  There's actually a couple of breaks that happened as a result of this branch's changes.

1. When you close and reopen the Tools dialog, the previous values of the Dice form go away.
2. because the section is nulled instead of hidden, the Math section is inaccessible from a normal width desktop browser.  This section is actually supposed to be visible side-by-side with the Dice section at L and XL widths.

Reply:
ok, I am going to take a look on this on master and compare to my branch

Reply:
In regards to number 2, on master I am not seeing anything side to side?

![image](https://user-images.githubusercontent.com/1933400/73061846-126f6580-3e69-11ea-9374-db03f9cdf4ad.png)